dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -35,6 +36,10 @@ import java.lang.NullPointerException
 import java.util.concurrent.TimeUnit
 
```

Comment:
Viele Anpassungen in dem file hier werden nicht mehr notwendig sein, sobald der `dataTypes` Filter wieder mandatory ist.

Reply:
Note an mich:  Rereview dann nochmal

Reply:
Sollten alle behoben sein.